cms/views.py

```diff
@@ -164,7 +164,7 @@ def details(request, slug):
     if page.login_required and not request.user.is_authenticated:
         return redirect_to_login(urlquote(request.get_full_path()), settings.LOGIN_URL)
 
-    content = page.get_title_obj(language=request_language)
+    content = page.get_title_obj(language=request_language, fallback=False)
```

Comment:
I guess with this the fallback option is probably never actually ever used now. I get why you're doing this though.

Reply:
Tbh I don't fully understand why you'd want a fallback in the first place. Either you get the content you requested or you don't. If there was a "fallback" it should happen through a redirect in my opinion, not through secretly replacing the content.

Reply:
The fallback option might be used elsewhere implicitly. I haven't checked everywhere.

Reply:
The fallback is for when you have a site in many languages there is a requirement that all of the pages didn't have to have a translation, it would serve a fallback. I'm not sure if this change would be accepted as this will change it's core behaviour:

"django CMS has a sophisticated multilingual capability. It is able to serve content in multiple languages, with fallbacks into other languages where translations have not been provided. It also has the facility for the user to set the preferred language and so on." 
http://docs.django-cms.org/en/latest/topics/multiple_languages.html